ABloom/ABloom/Presentation/CoreViews/Menu/SubViews/MyAccount/MyAccountView.swift

```diff
@@ -53,15 +67,61 @@ extension MyAccountView {
         HStack {
           Text("결혼까지 D-\(myAccountVM.dDay ?? 0)")
             .fontWithTracking(.footnoteR)
+            .foregroundStyle(.stone500)
           
           Spacer()
           
-          Text("정보 수정하기 >")
-            .fontWithTracking(.footnoteR)
+          Button {
+            myAccountVM.showActionSheet = true
+          } label: {
+            Text("정보 수정하기 >")
+              .fontWithTracking(.footnoteR)
+              .foregroundStyle(.stone500)
+          }
         }
         .foregroundStyle(.stone500)
```

Comment:
```suggestion
        .foregroundStyle(.stone500)
        .fontWithTracking(.footnoteR)
```
HStack 외부에 이렇게 작성해보는 건 어떨까요?! 적용 안될까요?!

Reply:
굿굿 수정 후 머지하겠습니다!